src/software/backend/simulation/physics/box2d_util.cpp

```diff
@@ -33,6 +33,16 @@ b2Vec2 createVec2(const Vector& vector)
     return ret;
 }
 
+Point createPoint(const b2Vec2& vec)
+{
+    return Point(vec.x, vec.y);
+}
+
+Vector createVector(const b2Vec2& vec)
+{
+    return Vector(vec.x, vec.y);
+}
+
```

Comment:
Just wondering why we have these helper functions if we can just call the Point or Vector constructor directly

Reply:
I just found it less tedious and error-prone